### ISSUE TITLE
Language Tour - mixins issue

### ISSUE THREAD

**Issue by ahan** [dart-lang/site-www#2227]
Page URL: https://dart.dev/guides/language/language-tour#adding-features-to-a-class-mixins

Found a typo? 
To specify that only certain types can use the mixin — for example, `so your mixin can invoke` a method that it doesn’t define — use on to specify the required superclass:

Description of issue:
Maybe it should be `so your mixin can't invoke`?

**Comment by kwalrath** [comment]
No, actually, it's _can_. If the mixin is guaranteed to have a certain superclass, then it can invoke a method from that superclass, even though the mixin doesn't define it.

Can you think of a way I can make it clearer?

**Comment by ahan** [comment]
Ok, I understood. I confused  it with Abstract Class. Thanks.

I not sure if this can be clearer: 
```
To specify that only certain types can use the mixin, use on to specify the required superclass. 
so your mixin, for example can invoke a method that it doesn’t define but define from required superclass.
```

**Comment by kwalrath** [comment]
I'd tweak the second sentence, but I do like making the first sentence simpler. We could have an example in the second sentence to make it clearer.

**Comment by kwalrath** [comment]
Actually, I'd like to keep this open, since I think the first sentence should be simpler.

**Comment by Adityaanand09** [comment]
hi! I would like to resolve this issue, since i am a beginner can you help me get started? Thank you.

**Comment by kwalrath** [comment]
> hi! I would like to resolve this issue, since i am a beginner can you help me get started? Thank you.

@Adityaanand09 I'd be happy for you to resolve this issue! I think all you need to do is go to the page (https://dart.dev/guides/language/language-tour), click the page icon at the upper right, and then edit the appropriate part of the page source using the GitHub UI. Then create a PR with an appropriate title and description, and (on a separate line in the description) `Fixes #2227`.

**Comment by bhumikasharma29** [comment]
Can I work on this?

**Comment by kwalrath** [comment]
> Can I work on this?

Absolutely! It's yours.

**Comment by bhumikasharma29** [comment]
Will you please review this?

**Comment by kwalrath** [comment]
Sorry for the delay, things got busy and this requires actual thought. :)

**Comment by malihanan** [comment]
Is the issue solved or can I work on it?

**Comment by kwalrath** [comment]
It's almost resolved. We have a fix (#2608), but I wanted @bhumikasharma29 to review my changes. It's been a week, though, so I might just merge without his review.
